catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/XmlModelBuilder.java

```diff
@@ -334,7 +346,7 @@ public void setValue(JAXBElement<?> valueNode) {
     @Override
     public JAXBElement<?> get() {
       notNull(propertyNode);
-      notNull(valueNode);
+      // notNull(valueNode);
```

Comment:
did you leave this comment here on purpose?

Reply:
Yeah I wanted it as part of the git history. The null guarantee had to be removed because of the `IS EMPTY` operator. Prior to that, null validation was possible.